package.json

```diff
@@ -165,11 +165,14 @@
     "invariant": "^2.2.4",
     "jest-fetch-mock": "^3.0.1",
     "jest-localstorage-mock": "^2.4.0",
+    "jquery": "^3.4.1",
     "lint-staged": "^8.1.5",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
+    "lodash-es": "^4.17.15",
     "lodash-webpack-plugin": "^0.11.5",
     "md5": "^2.2.1",
     "mini-css-extract-plugin": "^0.4.5",
+    "moment": "^2.24.0",
```

Comment:
Why was this added here? Are we actually using `moment` anywhere? I see it was in our code before, but I'm wondering why.

Reply:
Odd, I don't see it in use anywhere in our codebase. I'd say it's safe to remove entirely.